tests/ethpm/test_dependencies.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from ethpm import Package
+from ethpm.dependencies import Dependencies
+from ethpm.exceptions import UriNotSupportedError
+from ethpm.validation import validate_build_dependencies
+
+
+@pytest.fixture
+def dependencies(monkeypatch, standard_token_manifest, piper_coin_manifest):
+    monkeypatch.setenv(
+        "ETHPM_IPFS_BACKEND_CLASS", "ethpm.backends.ipfs.DummyIPFSBackend"
+    )
+    deps = piper_coin_manifest["build_dependencies"]
+    dep_pkgs = {dep: Package.from_ipfs(uri) for dep, uri in deps.items()}
+    return Dependencies(dep_pkgs)
+
+
+@pytest.fixture
+def safe_math_lib_package(safe_math_manifest):
+    return Package(safe_math_manifest)
+
+
+@pytest.fixture
+def invalid_manifest(piper_coin_manifest):
+    piper_coin_manifest["build_dependencies"]["_invalid_package_name"] = []
+    return piper_coin_manifest
+
+
+def test_dependencies_implements_getitem(dependencies, safe_math_lib_package):
+    assert isinstance(dependencies, Dependencies)
```

Comment:
I think we can either drop this assertion **or** move it into the `dependencies` fixture itself if you want to maintain the sanity check.